### ISSUE TITLE
emapper2gbk genomes issue

### ISSUE THREAD

**Issue by alsmadin01** [AuReMe/emapper2gbk#12]
* eggnog2gbk version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by ArnaudBelcour** [comment]
Hi @alsmadin01,

I think the issue comes from the empty lines in the `metagenome-name-final.tsv`. If I understand correctly your screenshots, this file is as follow:

|                               |                          |
|--------------------------|---------------------|
| MGYG000000002 | Blautia faecis   | 
|                               |                          |  
| MGYG000000003 | Alistipes shahii | 
|                               |                          |  

When emapper2gbk tries to read the file, it encounters the empty line between `MGYG000000002` and `MGYG000000003`. It tries to parse this line, expecting tabulations and 2 columns. But as it is empty it fails and returns the error you encounter.

Maybe by removing the empty lines, such as in this example:

|                               |                          |
|--------------------------|---------------------|
| MGYG000000002 | Blautia faecis   | 
| MGYG000000003 | Alistipes shahii | 

It could fix the issue.

Best Regards,
Arnaud Belcour.

**Comment by alsmadin01** [comment]
@ArnaudBelcour 
Yes, I actually figured that out yesterday and it worked. I created the tabulation in Excel and converted it to tsv, and I guess that leaves blank lines in between that I haven’t noticed or taken into consideration. Thank you!

**Comment by alsmadin01** [comment]
Hi @ArnaudBelcour,

As a follow up, can I run emapper2gbk for species not present in the EBI catalog like "UBA737 sp900554525" or "RC9"? I have gff, EggNOG annotation, fasta, and faa files for each. Also wanted to ask what kind of taxonomic information is extracted when species name is provided (like in a .tsv in my case).
